task2/src/main/java/ru/cft/focusstart/Main.java

```diff
@@ -0,0 +1,69 @@
+package ru.cft.focusstart;
+
+import ru.cft.focusstart.InputOutput.UserFileReader;
+import ru.cft.focusstart.ShapesClasses.*;
+
+import java.io.FileNotFoundException;
+
+import static ru.cft.focusstart.InputOutput.UserFileReader.*;
+
+public class Main {
+
+    private static String INPUT_FILE;
+    private static String OUTPUT_FILE;
+
+    private static void argsParser(String[] args) {
+        try {
+            INPUT_FILE = args[0];
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Параметры командной строки не указаны! (Нет входящего файла)");
+            System.exit(0);
+        }
+        try {
+            OUTPUT_FILE = args[1];
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Файла для сохранения не указано. Выводим с консоль\n");
+        }
+    }
+
+    public static void main(String[] args) {
+
+        argsParser(args);//Парсим аргументы командной строки
```

Comment:
В argsParser сложная логика с исключениями и глобальными переменными INPUT_FILE, OUTPUT_FILE
Все это  можно переписать как
```suggestion
       int count = args.length;
        if (count == 0) {
            System.out.println("Параметры командной строки не указаны! (Нет входящего файла)");
            System.exit(0);
        }
        String inputFileName = args[0];
        String outputFileName = null;
        if (count == 2) {
            outputFileName = args[1];
        }
```